src/bin/vip-wp.js

```diff
@@ -115,13 +115,17 @@ commandWrapper( {
 	envContext: true,
 	appQuery,
 } )
-	.argv( process.argv, async ( arg, opts ) => {
-		const isShellMode = 'shell' === arg[ 0 ];
-		const isSubShell = 0 === arg.length;
-		const cmd = arg.join( ' ' );
+	.argv( process.argv, async ( args, opts ) => {
+		const isShellMode = 'shell' === args[ 0 ];
+		const isSubShell = 0 === args.length;
+
+		// Have to re-quote anything that needs it before we pass it on
+		const quotedArgs = requoteArgs( args );
+
+		const cmd = quotedArgs.join( ' ' );
 
 		// Store only the first 2 parts of command to avoid recording secrets. Can be tweaked
-		const commandForAnalytics = arg.slice( 0, 2 ).join( ' ' );
```

Comment:
At some point we could mimic the sanitizeCommands logic here. But probably want to hold off on that until we're sure all the cases are covered